build/tasks/npmcopy.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const fs = require( "fs" );
+const path = require( "path" );
+
+const files = {
+	"bootstrap/bootstrap.css": "bootstrap/dist/css/bootstrap.css",
+	"bootstrap/bootstrap.min.css": "bootstrap/dist/css/bootstrap.min.css",
+	"bootstrap/bootstrap.min.css.map": "bootstrap/dist/css/bootstrap.min.css.map",
+
+	"core-js-bundle/core-js-bundle.js": "core-js-bundle/minified.js",
+	"core-js-bundle/LICENSE": "core-js-bundle/LICENSE",
+
+	"npo/npo.js": "native-promise-only/lib/npo.src.js",
+
+	"qunit/qunit.js": "qunit/qunit/qunit.js",
+	"qunit/qunit.css": "qunit/qunit/qunit.css",
+	"qunit/LICENSE.txt": "qunit/LICENSE.txt",
+
+	"requirejs/require.js": "requirejs/require.js",
+
+	"sinon/sinon.js": "sinon/pkg/sinon.js",
+	"sinon/LICENSE.txt": "sinon/LICENSE"
+};
+
+async function npmcopy() {
+	await fs.promises.mkdir( path.resolve( __dirname, "..", "..", "external" ), {
```

Comment:
It’d be easier to read & more portable if you defined root dir at the top in a similar way and then use it here & below instead of `__dirname` repeatedly. Especially once we migrate Node scripts to ESM as there’s no `__dirname` there.